pkg/api/pss.go

```diff
@@ -86,6 +82,16 @@ func (s *server) pssPostHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *server) pssWsHandler(w http.ResponseWriter, r *http.Request) {
+	upgrader := websocket.Upgrader{
+		ReadBufferSize:  swarm.ChunkSize,
+		WriteBufferSize: swarm.ChunkSize,
+		CheckOrigin: func(r *http.Request) bool {
+			if o := r.Header.Get("Origin"); o != "" && (s.CORSAllowedOrigins == nil || containsOrigin(o, s.CORSAllowedOrigins)) {
```

Comment:
Currently the tests are failing as this function returns false if `r.Header.Get("Origin") == ""`, which stops the handshake. If there are no "Origin" header, this function should return true `a per https://github.com/gorilla/websocket/blob/v1.4.2/server.go#L90.

Reply:
great, thanks!

yes, must run tests before pushing code 😂 

baby steps Dan 🤦